config.md

```diff
@@ -28,10 +28,12 @@ For example, if a configuration is compliant with version 1.1 of this specificat
 
 **`root`** (object, REQUIRED) specifies the container's root filesystem.
 
-* **`path`** (string, REQUIRED) Specifies the path to the root filesystem for the container.
+* **`path`** (string, OPTIONAL) Specifies the path to the root filesystem for the container.
   The path is either an absolute path or a relative path to the bundle.
-  On Linux, for example, with a bundle at `/to/bundle` and a root filesystem at `/to/bundle/rootfs`, the `path` value can be either `/to/bundle/rootfs` or `rootfs`.
-  A directory MUST exist at the path declared by the field.
+  On Linux, for example, with a bundle at `/to/bundle` and a root filesystem at `/to/bundle/rootfs`, the `path` value can be either `/to/bundle/rootfs` or `rootfs`. 
+  On Windows, for Windows Server Containers, this field is REQUIRED. For Hyper-V Containers, this field MUST NOT be supplied.
```

Comment:
> For writing validation tooling, how should runtime-tools distinguish between a Windows Server config and a Hyper-V config?

See the associated PR in https://github.com/opencontainers/runtime-spec/pull/818

Have updated the bundle requirements, the go type and the schema